db/migrate/20230307051200_add_position_to_sidebar_section_links.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddPositionToSidebarSectionLinks < ActiveRecord::Migration[7.0]
+  def change
+    add_column :sidebar_section_links, :position, :integer, default: 0, null: false
```

Comment:
I believe position needs to be unique one `sidebar_section_id` as well and perhaps `user_id`? We can't have multiple `sidebar_section_links` in a section for a given user to have the same position.

Reply:
I think we might be able to do an on conflict do nothing clause or something similar to the update all?

Reply:
Just checked that Postgres offers two option:
`ON CONFLICT DO NOTHING` - not great, as we want to update link to the new position
`ON CONFLICT DO UPDATE updates the existing row that conflicts with the row proposed for insertion as its alternative action.` - also not something we want.

I can hack it by calculating position to avoid conflict ones. So when original position was 1,2,3 - after reorder positions would be 4,5,6. With next reorder it would be back to 1,2,3 etc.